package.json

```diff
@@ -7,15 +7,18 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "src/index.js",
+  "main": "ui-kit.cjs.js",
+  "module": "ui-kit.esm.js",
   "scripts": {
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "precommit": "lint-staged",
     "prepublish": "node ./scripts/generate-icon-exports.js",
     "prebuild": "node ./scripts/generate-base-colors.js && node ./scripts/generate-base-shadows.js && node ./scripts/generate-colors-for-story.js",
-    "prestart": "node ./scripts/generate-base-colors.js && node ./scripts/generate-base-shadows.js && node ./scripts/generate-colors-for-story.js",
-    "build": "build-storybook -o .public",
-    "start": "start-storybook -p 9002",
+    "prestorybook:start": "node ./scripts/generate-base-colors.js && node ./scripts/generate-base-shadows.js && node ./scripts/generate-colors-for-story.js",
+    "storybook:build": "build-storybook -o .public",
```

Comment:
Cool, I'll push the netlify config file so we can change the build script from there.

Reply:
https://github.com/commercetools/ui-kit/pull/34/commits/5014336a36ac7857fdc82b461b5de69642f00f9b

Reply:
<img width="1174" alt="image" src="https://user-images.githubusercontent.com/1110551/45229942-9b631c80-b2c7-11e8-82ec-d850db3b9291.png">